src/form-element-creator.ts

```diff
@@ -0,0 +1,73 @@
+import { ElementCreator } from "./element-creator";
+
+export class FormElementCreator {
+    private elementCreator: ElementCreator;
+
+    constructor(elementCreator: ElementCreator) {
+        this.elementCreator = elementCreator;
+    }
+
+    createInput(form: HTMLFormElement, element: any): void {
+        const input = this.elementCreator.createElement('input') as HTMLInputElement;
+        this.setInputAttributes(input, element);
+        if (element.name === 'Phone') {
+            const select = this.elementCreator.createElement('select') as HTMLSelectElement;
+            this.createOption(select, 'PL', 'Poland (+48)');
+            this.createOption(select, 'US', 'United States (+1)');
```

Comment:
czy to ma sens pchac komus na sile dwa telefony? Lub checkbox z telefonami dwoma ? Lepiej zrobic liste z pojedynczym wyborem i tam zaciagnac przez jakies API liste telefonow calego swiata np